src/lib/tests/tests.cpp

```diff
@@ -11,9 +11,9 @@ TEST(TestX86Vec, Float128LoadScalar) {
   float abc[12] = {00.f, 01.f, 02.f, 03.f, 04.f, 05.f,
                    06.f, 07.f, 08.f, 09.f, 10.f, 11.f};
 
-  __m128 correct_x = _mm_setr_ps(00.f, 01.f, 02.f, 03.f);
```

Comment:
Did you change the behaviour of the scalar loader? from loading 4 contiguous to loading kinda SOA? I'm not sure what this will break but will investigate.

Reply:
Yeah so the above `abc[12]` is actually a 4x3 set of coordinates, so the correct x is every 3rd value.  I'm assuming that the user hasn't transposed their input already, partly because they don't know the width to transpose to (4 or 8).